client/my-sites/checkout/src/components/checkout-main.tsx

```diff
@@ -771,6 +803,8 @@ export default function CheckoutMain( {
 					showErrorMessageBriefly={ showErrorMessageBriefly }
 					siteId={ updatedSiteId }
 					siteUrl={ updatedSiteSlug }
+					isAkPro500Cart={ isAkismetProMultipleLicensesCart }
+					onChangeAkProQuantity={ changeAkismetPro500CartQuantity }
```

Comment:
I think we should try and avoid passing these values in so early and then passing them 2-3 more components. Would it be possible to determine these values from the component that uses them? (i.e. wp-order-review-line-items)

Reply:
Yes, I'll move them to `wp-order-review-line-items.tsx`.